manifests/g/geeksoftwareGmbH/PDF24Creator/11.11.0/geeksoftwareGmbH.PDF24Creator.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: geeksoftwareGmbH.PDF24Creator
+PackageVersion: 11.11.0
+InstallerLocale: en-US

```

```suggestion
InstallerLocale: en-US
Scope: machine
```
Package install machine-wide